packages/dnb-eufemia/src/extensions/forms/constants/locales/sv-SE.ts

```diff
@@ -0,0 +1,217 @@
+      removeButton: 'Ta bort',
+      editButton: 'Ändra',
+    },
+    IterateEditContainer: {
+      removeButton: 'Ta bort',
+      doneButton: 'Klar',
+      cancelButton: 'Avbryt',
+      errorInContainer: 'Felen ovan måste åtgärdas.',
+    },
+    IteratePushContainer: {
+      createButton: 'Lägg till',
+      itemsLimitReached: 'Du har nått gränsen på: {limit}',
+    },
+
+    /**
+     * Base fields
+     */
+    StringField: {
+      errorMinLength:
+        'Värdet kan inte vara kortare än {minLength} tecken.',
+      errorMaxLength: 'Värdet kan inte vara längre än {maxLength} tecken.',
+    },
+    NumberField: {
+      errorMinimum: 'Värdet måste vara minst {minimum}.',
+      errorMaximum: 'Värdet får vara högst {maximum}.',
+      errorExclusiveMinimum:
+        'Värdet måste vara större än {exclusiveMinimum}.',
+      errorExclusiveMaximum:
+        'Värdet måste vara mindre än {exclusiveMaximum}.',
+      errorMultipleOf: 'Värdet måste vara en multipel av {multipleOf}.',
+    },
+    BooleanField: {
+      yes: 'Ja',
+      no: 'Nej',
+    },
+    ToggleField: {
+      yes: 'Ja',
+      no: 'Nej',
+    },
+
+    /**
+     * Feature fields
+     */
+    PostalAddress: {
+      label: 'Postadress',
+      errorRequired: 'Du måste fylla i en postadress.',
+      errorPattern: 'Ogiltig postadress. Ange en postadress.',
+    },
+    StreetAddress: {
+      label: 'Gatuadress',
+      errorRequired: 'Du måste fylla i en gatuadress.',
+      errorPattern: 'Ogiltig gatuadress. Ange en gatuadress.',
+    },
+    Date: {
+      label: 'Datum',
+      errorRequired: 'Du måste ange ett giltigt datum.',
+      errorMinDate: 'Valt datum kan inte vara före {date}.',
+      errorMaxDate: 'Valt datum kan inte vara efter {date}.',
+      errorStartDateMinDate: 'Startdatum kan inte vara före {date}.',
+      errorStartDateMaxDate: 'Startdatum kan inte vara efter {date}.',
+      errorEndDateMinDate: 'Slutdatum kan inte vara före {date}.',
+      errorEndDateMaxDate: 'Slutdatum kan inte vara efter {date}.',
+    },
+    Expiry: {
+      label: 'Utgångsdatum',
+      errorMonth: '{month} är inte en giltig månad.',
+      errorYear: '{year} är inte ett giltigt år.',
+    },
+    Email: {
+      label: 'E-postadress',
+      errorRequired: 'Du måste fylla i en e-postadress.',
+      errorPattern: 'Ogiltig e-postadress. Ange en e-postadress.',
+    },
+    FirstName: {
+      label: 'Förnamn',
+      errorRequired: 'Du måste fylla i förnamn.',
+      errorPattern:
+        'Endast bokstäver och tecken som bindestreck och mellanslag är tillåtna.',
+    },
+    LastName: {
+      label: 'Efternamn',
+      errorRequired: 'Du måste fylla i efternamn.',
+      errorPattern:
+        'Endast bokstäver och tecken som bindestreck och mellanslag är tillåtna.',
+    },
+    CompanyName: {
+      label: 'Företagsnamn',
+      errorRequired: 'Du måste fylla i företagsnamn.',
+    },
+    NationalIdentityNumber: {
+      label: 'Födselsnummer (11 siffror)',
```

Comment:
```suggestion
      label: 'Personnummer (10 siffror)',
```

It's a bit problematic to change from 11 to 10 digits, as we then should also change the actual validation of the component 🤔

Reply:
The string content should be based on a country property, and not the locale. So we need to adjust this also for Norwegian actually.